app/helpers/braintree_view_helpers.rb

```diff
@@ -17,4 +17,28 @@ def braintree_transaction_link(payment_id)
       payment_id
     end
   end
+
+  ##
+  # Generate a client token for a user
+  # @return [String, NilClass]
+  #
+  def braintree_client_token_for(user = nil)
+    options = {}
+    options[:customer_id] = user.braintree_customer_id if user.present?
+
+    braintree_gateway.generate_client_token(options)
+  rescue => error
```

Comment:
I'm curious on thoughts about rescuing here. I rescue so that we don't return a 500 resulting in the user not getting a rendered page (Maybe braintree is down).